Moovies/Moovies/MainModule/Presentor/MainPresentor.swift

```diff
@@ -25,43 +26,39 @@ enum MoviesType {
     }
 }
 
+    // MARK: - MainViewPresentorProtocol
 protocol MainViewPresentorProtocol: AnyObject {
     var films: Film { get }
     func getMoviesOfType(_ type: MoviesType)
 }
 
+    // MARK: - MainPresentor
 class MainPresentor: MainViewPresentorProtocol {
+
+    // MARK: - Private Properties
+    private var movieAPIservice: MovieAPIServiceProtocol
     weak var view: MainViewProtocol?
     var films: Film
```

Comment:
```suggestion
    var films: Film
```
выше private подними